Katello/katello
# Fixes #28056: Use ForemanModal in Katello modals

Converting some existing Katello modals to the new ForemanModal component.

#### Depends on:
~~* [RM #27853 - Integrate modals with Redux store] [merged] (https://github.com/theforeman/foreman/pull/7105)~~

#### To test this PR:
~* IMPORTANT: In Foreman repo, check out branch `jeremylenz/27853-modals-redux`~
* ~Make sure your Foreman `package.json` lists `"@theforeman/vendor": "^2.15.7"`~
* In Foreman repo, run `npm install`
